test/first_party/CMakeLists.txt

```
@@ -63,7 +61,11 @@ if (GCC_BIN_RV64)
 endif()
 
 if (NOT CROSS_COMPILER_COMMAND_RV32 OR NOT CROSS_COMPILER_COMMAND_RV64)
-    message(FATAL_ERROR "No suitable cross-compiler found. We recommend downloading Clang from https://github.com/llvm/llvm-project/releases")
+    if (CLANG_BIN)
+        message(FATAL_ERROR "Your Clang compiler does not support RISC-V (see '${CLANG_BIN} -print-targets') and another suitable cross-compiler was not found. Please download one from https://github.com/llvm/llvm-project/releases")
```

```suggestion
        message(FATAL_ERROR "Your Clang compiler does not support RISC-V (see '${CLANG_BIN} -print-targets') and another suitable cross-compiler was not found.")
```

Otherwise we're repeating ourselves I think.

---

It uses `FATAL_ERROR` so will exit immediately after either message.

---

Ah yes sorry I didn't see that changed.